lib/client.js

```diff
@@ -174,10 +174,10 @@ class Metacom extends EventEmitter {
         const dest = new URL(this.url);
         const protocol = dest.protocol === 'ws:' ? 'http' : 'https';
         const url = `${protocol}://${dest.host}/api`;
-        const res = metautil.fetch(url, {
+        const res = await metautil.fetch(url, {
           method: 'POST',
           headers: { 'Content-Type': 'application/json' },
-          body: JSON.stringify(packet),
+          body: packet,
```

Comment:
`body` should be string, buffer of blob: https://developer.mozilla.org/en-US/docs/Web/API/fetch

```suggestion
          body: JSON.stringify(packet),
```